src/renderers/art/ReactART.js

```diff
@@ -93,7 +93,7 @@ function injectAfter(parentNode, referenceNode, node) {
 
 // ContainerMixin for components that can hold ART nodes
 
-const ContainerMixin = assign({}, ReactMultiChild, {
+const ContainerMixin = assign({}, ReactMultiChild.prototype, {
 
```

Comment:
This fixes flow errors introduced by #7649, so I was just wondering if we'd cut a release that contained some flow errors.